samples/snippets/pom.xml

```diff
@@ -57,7 +57,11 @@
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-pubsub</artifactId>
     </dependency>
-    
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-bigquery</artifactId>
+    </dependency>
+
```

Comment:
Either move this inside the `START/END_EXCLUDE` block, or add a comment for why the inclusion here. Everything inside the region tag `pubsub_install_with_bom` will be dynamically included on the published client lib quickstart:  https://cloud.google.com/pubsub/docs/publish-receive-messages-client-library#pubsub-client-libraries-java

Reply:
Done.